proto_test.go

```diff
@@ -12,109 +12,6 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-func TestRefType(t *testing.T) {
```

Comment:
Any reason to delete this test? It looks like we're still using `refType`?

Reply:
Oh whoops. That was not meant to be deleted 😕